stdlib/asm/crypto/stark/deep_queries.masm

```diff
@@ -127,22 +127,20 @@ end
 #! Loads the next query rows in the main, auxiliary and constraint composition polynomials traces.
 #! It takes a pointer to the current random query index and returns that index.
 #!
-#! Input: [query_ptr, ...]
+#! Input: [X, query_ptr, ...]
 #! Output: [index, query_ptr, ...]
```

Comment:
`X` is a word values of which are not important (because we'll overwrite them), right? If so, I'd add a brief comment explaining this.

Also, the `Output` description doesn't match stack state on line 275 below.

Reply:
Agreed, changed to Y and added an explanation.
Fixed the the output